launcher/net/PasteUpload.cpp

```diff
@@ -36,74 +36,81 @@
  */
 
 #include "PasteUpload.h"
-#include "Application.h"
-#include "BuildConfig.h"
 
-#include <QDebug>
-#include <QFile>
 #include <QHttpPart>
 #include <QJsonArray>
 #include <QJsonDocument>
 #include <QJsonObject>
+#include <QRegularExpression>
 #include <QUrlQuery>
 
-#include "net/Logging.h"
-
-std::array<PasteUpload::PasteTypeInfo, 4> PasteUpload::PasteTypes = { { { "0x0.st", "https://0x0.st", "" },
-                                                                        { "hastebin", "https://hst.sh", "/documents" },
-                                                                        { "paste.gg", "https://paste.gg", "/api/v1/pastes" },
-                                                                        { "mclo.gs", "https://api.mclo.gs", "/1/log" } } };
-
-PasteUpload::PasteUpload(QWidget* window, QString text, QString baseUrl, PasteType pasteType)
-    : m_window(window), m_baseUrl(baseUrl), m_pasteType(pasteType), m_text(text.toUtf8())
-{
-    if (m_baseUrl == "")
-        m_baseUrl = PasteTypes.at(pasteType).defaultBase;
-
-    // HACK: Paste's docs say the standard API path is at /api/<version> but the official instance paste.gg doesn't follow that??
-    if (pasteType == PasteGG && m_baseUrl == PasteTypes.at(pasteType).defaultBase)
-        m_uploadUrl = "https://api.paste.gg/v1/pastes";
-    else
-        m_uploadUrl = m_baseUrl + PasteTypes.at(pasteType).endpointPath;
-}
-
-PasteUpload::~PasteUpload() {}
-
-void PasteUpload::executeTask()
+const QVector<PasteUpload::RegReplace> PasteUpload::AnonimizeRules = {
+    RegReplace(QRegularExpression("C:\\\\Users\\\\([^\\\\]+)\\\\", QRegularExpression::CaseInsensitiveOption),
+               "C:\\Users\\********\\"),  // windows
```

Comment:
It's ok forward slashes as I do not change the rest of the path so I can not just replace them with backslashes as it would create something like: `C:/Users/******/Prismlauncers\instances\MyInstance...`.
Having mixed separators is confuzing